libnavigator/build.gradle

```diff
@@ -4,15 +4,17 @@ apply plugin: 'org.jetbrains.dokka'
 apply plugin: 'com.jaredsburrows.license'
 apply from: "${rootDir}/gradle/ktlint.gradle"
 
-dokka {
-    outputDirectory = "$buildDir/kdoc"
-    configuration {
-        moduleName = 'libnavigator'
-        reportUndocumented = true
+dokkaHtml {
+    outputDirectory.set(new File(buildDir.path, "kdoc"))
+    moduleName.set("LibNavigator")
```

Comment:
All of the classes in this module are internal (or in an internal package), can we remove this module from being published to the website?

Reply:
@LukasPaczos The same applicable for `Notification` and `Router` modules. Problem here that we don't have a mechanism to track if public api in modules appear without dokka. So here like a trade off: - we publish a whole module and keep tracking it - or remove it from dokka set and doesn't have automative mechanism.

Reply:
Can we not specify which modules should be published as part of dokka's multimodule task?

>  Problem here that we don't have a mechanism to track if public api in modules without dokka

Regardless, since all classes are internal or in internal packages in those modules, dokka task does not iterate over them anyway, right? So we should be fine to remove the task without actually losing any value.

Reply:
> Regardless, since all classes are internal or in internal packages in those modules, dokka task does not iterate over them anyway, right? So we should be fine to remove the task without actually losing any value.

I'm agree with it. For customers perspective it shouldn't shared at all 👍 


> Can we not specify which modules should be published as part of dokka's multimodule task?
actually we're not specify it outside of a module. Each module that use dokka plugin automatically consumed by dokka-multimodule task.

Reply:
not possible just remove modules that shouldn't be a part of api-docs, `publish` configuration plugin depends on dokka 
https://github.com/mapbox/mapbox-navigation-android/blob/5b325bb67ded6b74f8c690f0318713e455a589a2/gradle/publish.gradle#L15
let's merge as it is. I'll add it to ticket to split work